services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -311,6 +311,17 @@ public Builder select(Select select) {
             return this;
         }
 
+        /**
+         * Determines the attributes to be returned in the result. See {@link Select} for examples and constraints.
+         * If not found, returns {@link Select#UNKNOWN_TO_SDK_VERSION}.
+         * @param select the selection criteria as a string
+         * @return a builder of this type
+         */
+        public Builder select(String select) {
+            this.select = Select.fromValue(select);
```

Comment:
I think we need to change the `select` variable to string and pass it to the low level client to make it actually work. If customers pass an unsupported string, it'd be initialized as `UNKNOWN_TO_SDK_VERSION`